dev-libs/klibc/klibc-2.0.4-r2.ebuild

```diff
@@ -54,8 +54,10 @@ KEYWORDS="~alpha amd64 ~arm ia64 -mips ~ppc ~ppc64 ~sparc x86"
 SLOT="0"
 IUSE="debug test custom-cflags"
 
-DEPEND="dev-lang/perl"
-RDEPEND="${DEPEND}"
+COMMON_DEPEND="dev-lang/perl"
+DEPEND="${COMMON_DEPEND}
+	sys-devel/bc"
+RDEPEND="${COMMON_DEPEND}"
```

Comment:
Well, this is better, but uses an unnecessary variable.

You can switch the order so that RDEPEND appears first, then use ${RDEPEND} in DEPEND declaration.